packages/embark/src/cmd/cmd_controller.js

```diff
@@ -916,3 +923,10 @@ async function setupCargoAndWatcher(engine) {
 
   await engine.events.request2("watcher:start");
 }
+
+function checkPipelinePlugin(engine) {
```

Comment:
I think this check should be moved in config.ts, where, when we build the EmbarkConfig, we can check for `app`. That way, we wouldn't have to put the function call at multiple places. 
I know it will make it so that we'd show the warning on commands that don,t even build, but anyway, they'd need to install sooner than later

Reply:
IMO no because the config.ts should be just config files stuff and not dependencies